<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferCommodity</class>
 <widget class="QDialog" name="TransferCommodity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TransferCommodity</string>
  </property>
  <widget class="QComboBox" name="packageNumFrom">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择仓库编号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="inputInformation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入转移数量</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="packageLocationTo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择接受仓库</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="packageNumTo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择仓库编号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原仓库所在地：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原仓库编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>转移量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>现仓库所在地：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>现仓库编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;商品转移&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>转移商品编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commodityNum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入商品编号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>转移</string>
   </property>
  </widget>
  <widget class="QComboBox" name="packageLocationFrom">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择转移仓库</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
